src/test/java/net/sf/jabref/logic/l10n/LocalizationTest.java

```diff
@@ -0,0 +1,100 @@
+package net.sf.jabref.logic.l10n;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.StringJoiner;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.assertEquals;
+
+public class LocalizationTest {
```

Comment:
Some of the tests should have more expressive names which describe what they test (probably also without the `test` prefix). 
I also find that some assertion are hard to read/understand and in my opinion they would profit if they are written over multiple lines (so that occurring objects get a name :) )